swift/ios-shoppe-demo/ServiceManagers/FullStoryManager.swift

```diff
@@ -97,3 +98,57 @@ func fsModifyPrivacy(setting: PrivacySetting, views: UIView?...) {
 func fsIdentify(id: String, userInfo: [String: Any]) {
     FS.identify(id, userVars: userInfo)
 }
+
+
+// below are functions for demostrating conversions: revenue attribution events
+func makeId(len: Int) -> String {
+    let letters : NSString = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+    let length = UInt32(letters.length)
+
+    var randomString = ""
+
+    for _ in 0 ..< len {
+        let rand = arc4random_uniform(length)
+        var nextChar = letters.character(at: Int(rand))
+        randomString += NSString(characters: &nextChar, length: 1) as String
+    }
+
+    return randomString
+}
+
+func fsCreateEvent(event: Event, with product: Product?) {
```

Comment:
I would expect a function in Swift with the word `create` to produce an `Event` rather than mutate an existing `Event` instance.

Reply:
Maybe the names of these `fsCreatEvent` should be `fsPostEvent`? I can go back to make that change in the other areas, what do you think @nvzqz

Reply:
I updated the methods I added to be more specific of the actions(which hopefully conform with swift norm), I think @HaroldFS has a CH card for this function name specifically